mslib/msui/wms_control.py

```diff
@@ -474,6 +474,7 @@ def __init__(self, parent=None, default_WMS=None, wms_cache=None, view=None):
             self.wms_cache = None
 
         # Initialise date/time fields with current day, 00 UTC.
+        # To do Before refactoring to an aware datetime object add a test to verify the WMS part.
```

Comment:
Without a blank  ToDo, my IDE can list those. 

by PyCharm e.g. 
![without_a_blank](https://github.com/Open-MSS/MSS/assets/278728/b1001d36-4969-4576-aaf1-f4fed9276b59)

We have a repeating issue to look at ToDo's. - check still ToDo's - filing issues etc.